gallery_plugins/plugin_alphacoders.py

```diff
@@ -27,16 +27,19 @@ def redirect(source):
         indexed_page = cur_url + "&page=%d" % index
         print("Crawling " + indexed_page)
         indexed_source = urlopen_text(indexed_page)
-        links = re.findall('<img .*? data-src=\"(.*?)\" alt=\"HD Wallpaper.*?>',indexed_source)
+        links = re.findall('<a href="(big.php\?i=\d+)" title=', indexed_source)
         if links:
-            redirects += map(lambda x: x.replace('thumb-350-',''), links)
+            for link in links:
+                image_page = urlopen_text('https://' + identifier + '/' + link)
+                image_urls = re.findall('<a href="(http[s]?://images\d?\.alphacoders\.com/\d+/\d+\.[jpng]+)"', image_page)
```

Comment:
looks like this line will work for .jpg and .png, but not for .jpeg with an "e" in it.  What do you think of adding an optional "e" in the regex?